src/ppx_irmin/README.md

```diff
@@ -0,0 +1,99 @@
+# ppx_irmin
```

Comment:
can you move this at the toplevel and/or add a link from the main README.

Reply:
Moved to `./README_PPX.md` and added some cross-referencing in https://github.com/mirage/irmin/pull/919/commits/0656db82386bc41c3663e55a491940fa23abfa14.